### PR TITLE
 New package: nheko-0.6.4 

### LINKED ISSUE #12086: Package request: Nheko
https://github.com/Nheko-Reborn/nheko

### PR DESCRIPTION
**Pull request by not-chicken**
[ci skip]   

closes #12086

### CODE REVIEW

**Comment by not-chicken** [comment]
Whoops!, guess I should have put `[ci skip]` in commit message itself? .

Tested to compile for aarch64, armv7l and x86-glibc on x86-glibc host. 

Seems to run fine on my machine, travis will probably fail due to 40min time limit so need testers for other archs.

**Comment by Johnnynator** [comment]
It only needs ~30 minutes on Travis, so no need for `[ci skip]`

File: srcpkgs/lmdb++/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'lmdb++'
+pkgname=lmdb++
+version=0.9.14.0+20160229
+revision=1
```
Comment (daniel-eys) [inline srcpkgs/lmdb++/template]: As `version` hasn't changed, this shouldn't be revived with an older revision. This has already been on `revision=2`:
`[-] lmdb++-0.9.14.0+20160229_2 C++11 wrapper for the LMDB database library (removed package)`

Comment (not-chicken) [reply srcpkgs/lmdb++/template]: Changed `revision=2`

Comment (not-chicken) [reply srcpkgs/lmdb++/template]: Should have bumped it to 3? :thinking:

Comment (daniel-eys) [reply srcpkgs/lmdb++/template]: Yes

Comment (not-chicken) [reply srcpkgs/lmdb++/template]: done

**Comment by not-chicken** [comment]
The lint error. It's a false alarm, right?

**Comment by Johnnynator** [comment]
> The lint error. It's a false alarm, right?

Yep, the version is just 3 so it complains about every 3 in distfiles :)

**Pull request merged**